presto-mixin/README.md

```diff
@@ -0,0 +1,129 @@
+# Presto mixin
+
+The Presto mixin is a set of configurable Grafana dashboards and alerts.
+
+The Presto mixin contains the following dashboards:
+
+- Presto overview
+- Presto coordinator overview
+- Presto worker overview
+- Presto logs overview
+
+and the following alerts:
+
+- PrestoHighInsufficientFailures
+- PrestoHighTaskFailuresWarning
+- PrestoHighTaskFailuresCritical
+- PrestoHighQueuedTaskCount
+- PrestoHighBlockedNodes
+- PrestoHighFailedQueriesWarning
+- PrestoHighFailedQueriesCritical
+
+## Presto overview
+The Presto cluster overview dashboard provides details on integration status/alerts, workers/coordinators, error failures, data throughput, blocked nodes, and distributed bytes.
```

Comment:
I'd probably stick to just calling this 'Presto overview' as I don't see the 'cluster' being included elsewhere.